packages/contracts-bedrock/test/kontrol/README.md

```diff
@@ -67,6 +67,8 @@ First, generate a deployment summary contract from the deploy script in [`Kontro
 ./test/kontrol/scripts/make-summary-deployment.sh [container|local|dev]
 ```
 
+> [!NOTE]
+> To create a fault proof summary deployment, set `KONTROL_FP_DEPLOYMENT=true` and run the script `./test/kontrol/scripts/make-summary-deployment.sh`.
 The [`make-summary-deployment.sh`](./scripts/make-summary-deployment.sh) supports the same execution modes as `run-kontrol.sh` below.
```

Comment:
nit, need a blank line here to prevent this original sentence from being included in the note
```suggestion
> To create a fault proof summary deployment, set `KONTROL_FP_DEPLOYMENT=true` and run the script `./test/kontrol/scripts/make-summary-deployment.sh`.

The [`make-summary-deployment.sh`](./scripts/make-summary-deployment.sh) supports the same execution modes as `run-kontrol.sh` below.
```